core/utils/wp-import.php

```diff
@@ -0,0 +1,1350 @@
+<?php
```

Comment:
It's better to have a dedicated folder for all wp import/export-related files inside the utils